contracts/test/emergency/StarkNetValidator.test.ts

```diff
@@ -219,6 +219,38 @@ describe('StarkNetValidator', () => {
     })
   })
 
+  describe('#setSourceAggregator', () => {
+    describe('when called by non owner', () => {
+      it('reverts', async () => {
+        await expect(
+          starkNetValidator.connect(eoaValidator).setSourceAggregator(ethers.constants.AddressZero),
```

Comment:
nit: It's weird to connect a contract like this. 

I'd prefer if we'd explicitly deploy a new test account (once, in beforeAll), call it `alice` and connect `alice` in tests like this.

Reply:
not done yet (with last push)